messages/ownship/pilot_card.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Pilot Card",
+  "description": "Short description of an vessel model static information",
+  "required": ["vessel_model_id", "updated"],
+  "properties": {
+    "vessel_model_id": {
+      "type": "string",
+      "title": "Physical sensor id",
+      "desciption": "Id of the reporting physical sensor equipment"
+    },
+    "updated": {
+      "type": "string",
+      "format": "date-time",
+      "title": "Timestamp of last edit",
+      "description": "Date and time when the vessel model data where edited or updated."
+    },
```

Comment:
Hmm, yes, if this is supposed to be a "live" document, a field like this (or similar) might be a nice-to-have.